modules/moordyn/src/MoorDyn_Driver.f90

```diff
@@ -144,7 +144,7 @@ PROGRAM MoorDyn_Driver
    CALL CPU_TIME ( ProgStrtCPU )                                    ! Initial time (this zeros the start time when used as a MATLAB function)
    
 
-   CALL WrScr( ' MD Driver updated 2022-01-12')
+   CALL WrScr( ' MD Driver updated 2024-01-16')
```

Comment:
I'd recommend printing `TRIM(MD_ProgDesc%Date)` instead of hardcoding the date here.

Reply:
I made this change in my most recent commit. I also removed a call to DispCopyrightLicense in the driver because MoorDyn.f90 displays the license when called by the driver.